examples/source/user-consent/Server_Side_User_Consent_Flow.html

```diff
@@ -0,0 +1,115 @@
+  <style amp-custom>
+    :root {
+      --space-2: 1rem;   /* 16px */
+      --space-3: 1.5rem; /* 24px */
+    }
+    .consentPopup {
+      padding: var(--space-3);
+      margin: 0 auto;
+      background: #fff;
+      border-radius: 5px;
+      position: relative;
+      max-width: 700px;
+      width: 95%;
+    }
+    .consentPopup > h2, /* overwrite ABE style */
+    .consentPopup > * {
+      margin: 0;
+      margin-bottom: var(--space-2);
+    }
+    .consentPopup > button {
+      margin-right: var(--space-2);
+    }
+    .popupOverlay {
+      height: 100vh;
+      width: 100vw;
+      background: rgba(0, 0, 0, 0.7);
+      padding: 5% 0;
+    }
+    .dismiss-button {
+      position: absolute;
+      right: var(--space-3);
+      top: var(--space-2);
+      cursor: pointer;
+    }
+    #post-consent-ui {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      padding: var(--space-3);
+    }
+  </style>
+</head>
+<body>
+
+  <!-- ## Defining the consent flow -->
+  <!--
+    We define a basic server-side consent flow, similar to [this sample](/content/amp-dev/documentation/examples/documentation/Client_Side_User_Consent_Flow.html). The flow should only trigger when the servers response that `consentRequired` is true and there is not `consentStateValue` or `consentString` for this user.
+
+    When a user lands on the AMP page, amp-consent will check local storage for an existing consent state. 
+    If a consent state exists, it will unblock all components. Otherwise, it will wait for the response from `checkConsentHref` (because `consentRequired` is set to 'remote') to determine if consent is required/if there is an existing consent state. If there is an existing consent state, amp-consent will use that value and unblock. Otherwise, it will show the `promptUi`.
+
+    A recommended design for sites that can manage their own consent (via external page) is to respond with the following response: 
+    ```json
+    {
+      "consentRequired": true,
+      "consentStateValue": "accepted"/"rejected"/"unknown",
+      "consentString": "example-string"
+    }
+    ```
+    Tip: Use [`expireCache: true`](https://amp.dev/documentation/components/amp-consent/?format=websites#expireCache), in your response to clear localStorage consent decision if you need to update it for syncing purposes. Additonally, [`onUpdateHref`](https://amp.dev/documentation/components/amp-consent/?format=websites#onupdatehref) can be used when users are managing their consent directly on the page.
+
+  -->
+  <amp-consent id="myUserConsent" layout="nodisplay">
+    <script type="application/json">{
+      "consentInstanceId": "server-side-consent",
+      "promptUI": "myConsentFlow",
```

Comment:
Can we not provide `promptUI` and `postPromptUI` in this example because it's supposed to be handled by server. 
But I noticed you put `onUpdateHref` which is kind of unnecessary without prompt, please let me know what do you think? Would there be a good use case to collect consent from server while also from client side prompt?

Reply:
No, you're right. I was trying to show all the fields that could be configured, instead of a more realistic example.